core/templates/dev/head/profile/preferences.html

```diff
@@ -115,28 +109,47 @@ <h2 class="oppia-page-heading">Preferences</h2>
                 <select2-dropdown choices="LANGUAGE_CHOICES"
                   item="$parent.preferredLanguageCodes" allow-multiple-choices="true"
                   new-choice-regex=".^" width="400"
-                  placeholder="Select preferred languages..."
+                  placeholder="<['I18N_PREFERENCES_SELECT_EXPLORATION_LANGUAGE' | translate]>"
                   on-selection-change="savePreferredLanguageCodes($parent.preferredLanguageCodes)">
                 </select2-dropdown>
               </div>
-              <span class="help-block" style="font-size: smaller">
-                These languages will be selected by default when you search the
-                gallery for explorations.
-              </span>
+              <span class="help-block" style="font-size: smaller" translate="I18N_PREFERENCES_PREFERRED_EXPLORATION_LANGUAGE_EXPLAIN"></span>
             </div>
           </div>
         </div>
 
+        <div role="form" class="form-horizontal">
+          <div class="form-group">
+            <label class="col-lg-2 col-md-2 col-sm-2" translate="I18N_PREFERENCES_PREFERRED_SITE_LANGUAGE"></label>
+            <div class="col-lg-10 col-md-10 col-sm-10">
+              <!--
+                If the ng-if is omitted, the select2-dropdown directive won't be
+                updated after the initial page load.
+              -->
+              <div ng-if="hasPageLoaded">
+                <select2-dropdown choices="SITE_LANGUAGE_CHOICES"
```

Comment:
For two reasons:
-- Consistency with the other language selector.
-- Because I image it having a lot of different languages, and then the automatic search of select2 can be very useful.
Let me know if you want me to change it!